solr/CHANGES.txt

```diff
@@ -335,6 +335,8 @@ Other Changes
   collections/collectionName into its state.json.  For many-collection clusters, this is an
   optimization when the cluster status is fetched.  (Nazerke Seidan, David Smiley)
 
+* SOLR-15461: SOLR-15461: Upgrade Apache Calcite to 1.27.0. (Mark Miller, Timothy Potter)
```

Comment:
Do we need to document anything about backwards compatibility? Or call out new features in the ref guide